src/main/java/kr/sesac/aoao/server/diary/repository/DiaryEntity.java

```diff
@@ -0,0 +1,71 @@
+package kr.sesac.aoao.server.diary.repository;
+
+import java.time.LocalDateTime;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import kr.sesac.aoao.server.diary.controller.dto.request.DiaryCreateRequest;
+import kr.sesac.aoao.server.global.entity.BaseEntity;
+import kr.sesac.aoao.server.global.exception.ApplicationException;
+import kr.sesac.aoao.server.todo.exception.TodoFolderErrorCode;
+import kr.sesac.aoao.server.todo.repository.PaletteEntity;
+import kr.sesac.aoao.server.user.repository.UserEntity;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "diary")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+public class DiaryEntity extends BaseEntity {
```

Comment:
```suggestion
@Getter
@Entity
@Table(name = "diary")
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class DiaryEntity extends BaseEntity {
```

setter 를 열어두면 값의 변경을 열어둔다는 의미가 있어 되도록이면 setter 사용은 지양하는 것이 좋습니당 특히나 여기선 롬복으로 처리하기 때문에 어디서 사용되는지 추적하기 더 어려워서 안쓰는편이 좋을 것 같아요!! 만약 변경이 필요하다면 필요한 값에 한해서만 updateXX() 등의 다른 이름으로 메서드를 만들어서 사용하는 편입니다!